airflow/settings.py

```diff
@@ -43,6 +42,48 @@
 SIMPLE_LOG_FORMAT = '%(asctime)s %(levelname)s - %(message)s'
 
 
+def log_to_stdout():
+
+    root_logger = logging.getLogger()
+
+    # default log level if not set externally (e.g. with --logging-level=DEBUG)
+    if root_logger.level == logging.NOTSET:
+        root_logger.setLevel(LOGGING_LEVEL)
+
+    for handler in root_logger.handlers:
+        if isinstance(handler, logging.StreamHandler):
+            root_logger.warn("not adding a stream handler: already present")
+            return
+
+    logformat = logging.Formatter(
+        "%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+
+    ch = logging.StreamHandler(sys.stdout)
+    ch.setFormatter(logformat)
+    root_logger.addHandler(ch)
+
+
+class WithLogger(object):
+    """
+    Convenience super-class to have a logger configured with the class name
+    """
+
+    def logger(self):
+        return logging.getLogger(self.__class__.__name__)
```

Comment:
Should the logger object be memoized somehow, otherwise getLogger is called at every logging event...